### PR TITLE
#BAH-921 | Refactor Bahmni IE Apps to migrate API-related classes to the api folder

### PR DESCRIPTION
**Pull request by chinzou**
https://bahmni.atlassian.net/browse/BAH-921

### CODE REVIEW

File: api/src/main/java/org/bahmni/customdatatype/datatype/FileSystemStorageDatatype.java [hunk 1]
```diff
@@ -9,34 +9,38 @@
 
 public class FileSystemStorageDatatype extends SerializingCustomDatatype<String> {
 
-    public static final String DEFAULT_ENCODING = "utf-8";
-    private String fileName;
```
Comment (rbuisson) [inline api/src/main/java/org/bahmni/customdatatype/datatype/FileSystemStorageDatatype.java]: @chinzou , did you really make any modification, other than format, on this file? Maybe you could un-stage those before commit?

Comment (chinzou) [reply api/src/main/java/org/bahmni/customdatatype/datatype/FileSystemStorageDatatype.java]: @rbuisson , there's no modification other than format. yes, i will un-stage it :)

File: api/src/test/java/org/bahmni/customdatatype/datatype/FileSystemStorageDatatypeTest.java [hunk 2]
```diff
@@ -13,55 +13,55 @@
 
 public class FileSystemStorageDatatypeTest {
 
-    FileSystemStorageDatatype fileSystemStorageDatatype;
+	FileSystemStorageDatatype fileSystemStorageDatatype;
```
Comment (rbuisson) [inline api/src/test/java/org/bahmni/customdatatype/datatype/FileSystemStorageDatatypeTest.java]: @chinzou , could you unstage this file too?

**Comment by rbuisson** [comment]
Thanks @chinzou , merged.

**Pull request merged**